deploy/heroku/main.tf

```
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    heroku = {
+      source  = "heroku/heroku"
+      version = "~> 4.0"
```

@hoangmirs Should we make it to the latest version?

```suggestion
      version = "~> 4.1"
```